front/app/api/analysis_heat_map_cells/useAnalysisHeatmapCells.ts

```diff
@@ -45,6 +45,8 @@ const useAnalysisHeatmapCells = (params: IAnalysysHeatmapCellsParams) => {
   >({
     queryKey: analysesKeys.list(params),
     queryFn: () => fetchAnalysisHeatmapCells(params),
+    refetchInterval: 1000 * 60, // 1 minute
```

Comment:
Maybe we can even request less often here.. wdyt?

Reply:
I think it's okay. Once a minute won't create too much stress on anything, and it's nice to have them on screen when we have them. (mostly auto tagging)